src/MicrosoftTeams.ts

```diff
@@ -12,6 +12,8 @@ namespace microsoftTeams
 {
     "use strict";
 
+    const version = "0.2";
```

Comment:
How are we going to rev this going forward? Also, how are you planning to publish this to v2 of our SDK without pulling in all of my authentication changes?